tensorflow/core/kernels/mkl_conv_ops.cc

```diff
@@ -643,6 +772,51 @@ class MklConvOp : public OpKernel {
 
       // Check whether src and filter need to be reordered
       Tinput* src_data = nullptr;
+#ifdef ENABLE_MKLDNN_V1
```

Comment:
This is also a big blob that's mostly similar with the original code. I think we can get rid of this by defining a few more things.

Reply:
Agree. Done.